ceph/rbd/initial_config.py

```diff
@@ -534,13 +540,13 @@ def initial_mirror_config(**kw):
         kw["ceph_cluster"] = cluster
         if cluster_name == ceph_cluster.name:
             primary_config = ret_config[cluster_name] = initial_rbd_config(
-                is_secondary=False, **kw
+                is_secondary=False, is_mirror=True, **kw
```

Comment:
By default what value `is_mirror` will be set?

Reply:
By default the parameter will not exist. We need to set it to true only if its a mirror configuration.